pipeline/rhos_scripts/quota_stats.groovy

```diff
@@ -281,11 +281,15 @@ def sendEmail(def quota_detail){
     */
     def body = readFile(file: "pipeline/vars/emailable-report.html")
     body += "<body><u><h1>RHOS-D Quota Usage Summary</h1></u><br />"
-    body += '''<p>Hi Team,
+    body += '''<p>Hi Team,<br /><br />
                 Please go through the instances created by each one of you and
-                please plan to clear unused instances in each project.</p><br />'''
+                please plan to clear unused instances in each project.<br />
+                RHOS-D URL: https://rhos-d.infra.prod.upshift.rdu2.redhat.com</p>'''
     body += fetch_html_table(quota_detail["Project Stats"], [80, 50], "Projects", "RAM usage in %")
     body += fetch_html_table(quota_detail["User Stats"], [20, 10], "Users", "Instance Count")
+    body += """\
+            <br /><p style="color:red;">NOTE: Reach out to "mgowri@redhat.com"
```

Comment:
@veera-raghava-reddy could we have a pdl instead of a specific user email address? This would allow us to maintain the list of notifiers outside and reduce the number of times we need to modify here.